packages/aws-cdk/lib/os.ts

```diff
@@ -30,7 +36,10 @@ export async function shell(command: string[], options: ShellOptions = {}): Prom
       stdout.push(chunk);
     });
 
-    child.once('error', reject);
+    child.once('error', (err) => {
+      debug('error', err);
+      reject(err);
```

Comment:
I believe this should result in an exception so not sure `debug` is needed before, but that's fine

Reply:
I added it as a debug, will remove.